frontend/src/app/testResults/TestResultsList.tsx

```diff
@@ -508,6 +518,20 @@ export const DetachedTestResultsList = ({
                   defaultSelect
                   onChange={setFilterParams("role")}
                 />
+                {validFacilities && validFacilities.length > 1 ? (
```

Comment:
Awesome, I like that it only appears when the user has access to more than one facility!

Reply:
Credit goes to @kenieh for that one :)